modules/uncertainty/R/ensemble.R

```diff
@@ -448,9 +455,9 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
                            stop.time =restart$stop.time, 
                            settings = settings,
                            new.state = new.state[i, ], 
-                           new.params = new.params[[i]], 
+                           new.params = new.params[[i]], #new.params$`646`[[i]] for debugging
```

Comment:
still need this comment here?

Reply:
I think the comment is helpful because when debugging you have to add in the extra dollar sign otherwise the function will return a misleading error, I suspect this has to do with the difference from calling a function versus running the function line by line